components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -94,6 +94,7 @@ type Config struct {
 
 	KymaVersion                          string
 	EnableOnDemandVersion                bool `envconfig:"default=false"`
+	DisableEvaluationProfile             bool `envconfig:"default=true"`
```

Comment:
it is not disabling evaluation profile so please change the name or change the configuration (like number of nodes per evaluation...)